tests/resources/example_docker_project/MLproject

```diff
@@ -9,4 +9,4 @@ entry_points:
   test_tracking:
     parameters:
       use_start_run: bool
-    command: "python docker_tracking_test.py {use_start_run}"
+    command: "python example_docker_project/docker_tracking_test.py {use_start_run}"
```

Comment:
Nit: maybe rename the `example_docker_project` directory to e.g. `scripts`?  It's a little confusing to have nested directories both named example_docker_project

Reply:
Actually, I'll just push a commit to address this - looks great otherwise!